.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,92 @@
+  - type: input
+    id: search_terms
+    attributes:
+      label: "🔎 Search Terms"
+      description: |
+        What search terms did you use when trying to find an existing bug report?
+
+        List them here so people in the future can find this one more easily.
+    validations:
+      required: true
+  - type: textarea
+    id: version_info
+    attributes:
+      label: "🕗 Version & Regression Information"
+      description: |
+        When did you start seeing this bug occur?
+
+        "Bugs" that have existed in TS for a long time are very likely to be FAQs; refer to
+        https://github.com/Microsoft/TypeScript/wiki/FAQ#common-bugs-that-arent-bugs
+
+
+        If possible, please try testing the nightly version of TS to see if it's already been fixed. For npm: `typescript@next`
+
+        This is also the 'Nightly' version in the playground: http://www.typescriptlang.org/play/?ts=Nightly
+  
+
+        Note: The TypeScript Playground can be used to try older versions of TypeScript.
+
+        Please keep and fill in the line that best applies.
+      value: |
+        - This is a crash
+        - This changed between versions ______ and _______
+        - This is the behavior in every version I tried, and I reviewed the FAQ for entries about _________
+        - I was unable to test this on prior versions because _______
+      render: Markdown
+    validations:
+      required: true
+  - type: input
+    id: playground_link
+    attributes:
+      label: ⏯ Playground Link
+      description: |
+        A link to a TypeScript Playground "Share" link which shows this behavior.
+
+        This should have the same code as the code snippet below, and use whichever settings are relevant to your report.
+
+
+        As a last resort, you can link to a repo, but these will be slower for us to investigate.
+      placeholder: "Playground link with relevant code: https://www.typescriptlang.org/play?#code/PTAEFkE9QYwewCYFNQHM5IM6gBZIE5JA"
+    validations:
+      required: false
+  - type: textarea
+    id: code
+    attributes:
+      label: 💻 Code
+      description: |
+        Please post the relevant code sample here as well.
+        This code and the Playground code should be the same, do not use separate examples.
+
+        ```ts
+        // We can quickly address your report if:
+        //  - The code sample is short. Nearly all TypeScript bugs can be demonstrated in 20-30 lines of code!
+        //  - It doesn't use external libraries. These are often issues with the type definitions rather than TypeScript bugs.
+        //  - The incorrectness of the behavior is readily apparent from reading the sample.
+        // Reports are slower to investigate if:
+        //  - We have to pare too much extraneous code.
+        //  - We have to clone a large repo and validate that the problem isn't elsewhere.
+        //  - The sample is confusing or doesn't clearly demonstrate what's wrong.
```

Comment:
This can just be regular text; it doesn't need to be a code block. If you drop the `// ` prefix, it should render okay.

Right now, the text is very small: https://github.com/jakebailey/ts-issue-template-testing/issues/new?assignees=&labels=&projects=&template=bug_report.yml

Reply:
Ironically it's bigger than the surrounding text 😅, but yeah, it doesn't look that great as a codeblock.

<img width="757" alt="image" src="https://github.com/microsoft/TypeScript/assets/4097841/638f03d7-9e82-4fc1-8cf0-75896840ec8d">

Reply:
I'll remove the code block and the `//`.